rpc/mock-amqp-channel_test.go

```diff
@@ -0,0 +1,92 @@
+// Automatically generated by MockGen. DO NOT EDIT!
+// Source: rpc/amqp-rpc-connection.go
```

Comment:
nit: I think the time for dashes in the names of files is over. It's not go style, and looks weird, especially when next to _test.go. I know we do this currently, but we have to eventually stop. Why not here?